docs/quickentity-editor/gamefileextensions.md

```diff
@@ -0,0 +1,117 @@
+- Add automatic patch loading
+- Display a graph view for pins
+- Display previews for what entities are referenced in external entities
+- Show the names of entities in the overrides view
+- Add text next to a repository ID, showing you which repo item is being referenced
+- Make obfuscated material properties human readable
+
+As an example of the last point, here's how a material override entity might look:
+
+```json
+{
+	"parent": "71625c2a5b536117",
+	"name": "shoes_sneaker_high",
+	"factory": "[assembly:/_pro/characters/assets/individuals/bulldog/privateinvestigator/materials/shoes_sneaker_high.mi].pc_entitytype",
+	"blueprint": "[assembly:/_pro/characters/assets/individuals/bulldog/privateinvestigator/materials/shoes_sneaker_high.mi].pc_entityblueprint",
+	"properties": {
+		"Clients": {
+			"type": "TArray<SEntityTemplateReference>",
+			"value": [
+				"edcfc381a1fcc7c6"
+			]
+		},
+		"Texture2D_01_enab": {
+			"type": "bool",
+			"value": true
+		},
+		"Diffuse_Color_01_Value_op": {
+			"type": "IRenderMaterialEntity.EModifierOperation",
+			"value": "eLeave"
+		},
+		"Texture2D_01": {
+			"type": "ZRuntimeResourceID",
+			"value": {
+				"resource": "[assembly:/_pro/characters/assets/individuals/bulldog/privateinvestigator/textures/shoes_sneaker_high.texture?/specular_a.tex](ascolormap).pc_tex",
+				"flag": "5F"
+			}
+		},
+		"ConstantVector1D_01_Value": {
+			"type": "float32",
+			"value": 5
+		},
+		"ConstantVector1D_01_Value_op": {
+			"type": "IRenderMaterialEntity.EModifierOperation",
+			"value": "eReplace"
+		},
+		"Diffuse_Color_01_Value": {
+			"type": "SColorRGB",
+			"value": "#ffffff"
+		},
+		"ConstantVector1D_05_Value": {
+			"type": "float32",
+			"value": 1
+		},
+		"ConstantVector1D_04_Value_op": {
+			"type": "IRenderMaterialEntity.EModifierOperation",
+			"value": "eReplace"
+		},
+		"ConstantVector1D_04_Value": {
+			"type": "float32",
+			"value": 0.2800000011920929
+		},
+		"ConstantVector1D_06_Value_op": {
+			"type": "IRenderMaterialEntity.EModifierOperation",
+			"value": "eReplace"
+		},
+		"ConstantVector1D_05_Value_op": {
+			"type": "IRenderMaterialEntity.EModifierOperation",
+			"value": "eReplace"
+		},
+		"m_eidParent": {
+			"type": "SEntityTemplateReference",
+			"value": "edcfc381a1fcc7c6"
+		}
+	}
+}
+```
+
+Diffuse_Color_01 is simple enough but what the hell is a ConstantVector1D_01? You can certainly experiment through trial and error in figuring out what all of these values do by twiddling knobs. But it's extremely time consuming to alter a value, apply the mod, boot up the game and see what difference you can see with your eyes. To make matters worse, ConstantVectors mean different things depending on what `MATE` the material uses.
+
+Game file extensions will show a friendly human-readable name next to the property, like so:
+
+![showcasing human readable names of properties](pathname:///media/quickentity-editor/gamefileextensions/gfe.png)
```

Comment:
Remove the `pathname://`

Reply:
You sure that's necessary? Worked locally and my other PR also had that and works live.

Reply:
where did that even come from? i'm fairly certain that shouldn't work

Reply:
I just copied the way someone else had done image embeds. Searching for "pathname:///" I'm getting 32 hits in 12 files. You tell me how that works :V

Reply:
I was doing it to avoid images getting randomly generated paths but I was thinking about redoing all the images properly at some point.

Reply:
@Notexe the generated paths are just content hashes, they don't get changed unless webpack does

Reply:
I'll fix it up

Reply:
Alright fixed. Pathname:// will need to be removed and the image in this PR should go into an img folder instead (like static/img/quickentity-editor/gamefileextensions/gfe.png)

And then just reference the image like: `![showcasing human readable names of properties](/img/quickentity-editor/gamefileextensions/gfe.png)`